syncano/models/fields.py

```diff
@@ -187,6 +187,19 @@ def to_python(self, value):
             raise self.ValidationError('Invalid value. Value should be an integer.')
 
 
+class ReferenceField(WritableField):
```

Comment:
cool but more DRY solution would be to inherit from `IntegerField` and overwrite `to_python` method :)